app/services/slugs/update_sluggable_history.rb

```diff
@@ -0,0 +1,19 @@
+class Slugs::UpdateSluggableHistory
+  def self.call(*args)
+    new(*args).call
+  end
+
+  def initialize(slug_prefix:, resource:)
+    @slug_prefix = slug_prefix
+    @resource = resource
+  end
+
+  def call
+    @resource.slugs.active.update_all(active: false) &&
+    @resource.slugs.active.build(
```

Comment:
Unable to call `#create` here as the parent hasn't been saved yet, reverted to `#build`